content/Certification/certifying_a_product/development.md

```diff
@@ -0,0 +1,197 @@
+5.7.6 and section 13.6 of the specification. These requirements are attested at
+certification in the
+
+[Matter Setup Code Guidelines](https://groups.csa-iot.org/wg/members-all/document/27481)
+
+### Multi-product QR codes
+
+For products where multiple devices are in one box, it can be challenging for
+users to properly match individual on-boarding materials to the physical
+products in the box. In such situations, manufacturers should follow the
+guidance in the [Multi Device Setup Codes Best Practices Guide]().
+
+### In-field updates to Matter
+
+If the software being developed is for a product already in the field, the
+Matter firmware will need to distributed as an over-the-air update using the
+current update mechanism on the device. In-field update considerations are
+discussed in section 5.8 of the specification, but it is important to note that
+in-field products are subject to the same per-unit requirements as Out-of-Box
+Natter products, and thus it is important to consider how to distribute not only
+the firmware, but also how to display the on-boarding material or how to do a
+background update using the current connected application and how to provision
+the per-unit materials discussed earlier in this section in a secure manner
+using the security materials available to the current firmware.
+
+## Certification programs
+
+Product developers should be aware of the various certification programs and how
+they may impact the product development process across the entire line.
+
+In particular:
+
+-   Products considering software component certification should understand
+    which types of products and operating environments qualify for this program.
+-   Products with various SKUs and variants should understand the portfolio
+    certification program and what constitutes a viable portfolio parent.
+-   Bridge devices should understand the bridge certification program and how
+    certification of the supported device types works.
+
+Product manufacturers should be aware that the CSA has a sunset policy in effect
+for prior specification revisions. The sunset policy is available in full at
+(TODO: where's the sunset policy?).
+
+## Distribution of Over-the-air updates (OTA) and re-certification
+
+As development continues on the Matter SDK, developers continue to find and fix
+bugs, improve performance and maintainability. Ongoing specification development
+works to improve interoperability and performance with new features. For these
+reasons, its important to consider delivering software updates using either the
+Matter OTA mechanism or a manufacturer-specific mechanism.
+
+The mechanism for delivering OTAs is an important product consideration that
+needs to be built into the shipping firmware. Update cadence affect the
+certification planning for the product since all updates need to be
+re-certified.
+
+[//]: #(TODO: do we want to talk about factory shipping image / 0-day split?
```

Comment:
Maybe a question for RS re: proposals around OTA during commissioning.

Reply:
Note: today, Matter requires that the software on the FSI is the certified image, it made no provisions for pre-certifying the product, or for doing an OTA prior to completing the OOBE or prior to completing device attestation.